azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/SetupFragment.kt

```diff
@@ -20,11 +24,12 @@ import com.azure.android.communication.ui.presentation.fragment.setup.components
 import com.azure.android.communication.ui.presentation.navigation.BackNavigation
 
 internal class SetupFragment(
```

Comment:
can we remove constructor?

Reply:
yes, removed the () from call and setup fragment.